images/runtime/node/README.md

```diff
@@ -14,13 +14,11 @@ ones used in the official Node image for each particular version.
 # Generate images from template
 
 Since our node images are identical except for the base image, which determines the node version,
-some of our node images are generated from a template file, `./Dockerfile.template`, which contains the placeholder
-for the base image name. File `./nodeVersions.txt` has a list of the supported node versions as their tags for the official node images.
+our final node images are generated from a template file, `./Dockerfile.template`, which contains the placeholder
```

Comment:
isn't it wrong for some of the images?

Reply:
not anymore, since we're rebuilding them all from scratch now.